test/jdk/java/awt/TrayIcon/DisposeInActionEventTest/DisposeInActionEventTest.java

```diff
@@ -100,30 +97,30 @@ public void start() {
         g.fillRect(6, 6, 20, 20);
         g.dispose();
 
-        systemTray = SystemTray.getSystemTray();
-        trayIcon = new TrayIcon(img);
+        SystemTray systemTray = SystemTray.getSystemTray();
+        TrayIcon trayIcon = new TrayIcon(img);
         trayIcon.setImageAutoSize(true);
         trayIcon.addActionListener(ev -> {
-            Sysout.println(ev.toString());
+            textArea.append(ev.toString());
```

Comment:
Similar changes to other places where textArea.append() is used.

```suggestion
            textArea.append(ev.toString() + "\n");
```